_posts/2020-03-29-newpipe-0.19.0-released.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title: "NewPipe 0.19.0 released: Folder your feeds and mute them"
+short: "NewPipe 0.19.0 released"
+date: 2020-03-29 00:00:00 +0100
+categories: [release, pinned]
+author: Poolitzer
+image: newpipe
+excerpt_separator: <!-- more -->
+---
+
+A new day, a new full release. In today's update, NewPipe is proudly bringing you a great way to sort your subscriptions and a mute button, on top of the usual smaller improvements and fixes<!-- more -->
```

Comment:
```suggestion
A new day, a new full release. In today's update, NewPipe is proudly bringing you a great way to sort your subscriptions, and a mute button, on top of the usual smaller improvements and fixes<!-- more -->
```

Reply:
the and is a listing here of only two items, the comma would be incorrect

Reply:
There is ambiguity, though. You could change it to "bringing you a mute button and a great way to sort your subscriptions".

Though I think that should be "manage your subscribed channels" just to be clear.

Reply:
"bringing you a mute button and a great way to manage your subscribed channel".

sounds good to me